db/indexes.go

```diff
@@ -98,14 +117,22 @@ var (
 		IndexRoleAccess: "ALL (ARRAY (op.name) FOR op IN OBJECT_PAIRS($sync.role_access) END)",
 		IndexChannels: "ALL (ARRAY [op.name, LEAST($sync.sequence,op.val.seq), IFMISSING(op.val.rev,null), IFMISSING(op.val.del,null)] FOR op IN OBJECT_PAIRS($sync.channels) END), " +
 			"$sync.rev, $sync.sequence, $sync.flags",
-		IndexAllDocs:    "$sync.sequence, $sync.rev, $sync.flags, $sync.deleted",
-		IndexTombstones: "$sync.tombstoned_at",
-		IndexSyncDocs:   "META().id",
+		IndexAllDocs:                "$sync.sequence, $sync.rev, $sync.flags, $sync.deleted",
+		IndexTombstones:             "$sync.tombstoned_at",
+		IndexSyncDocs:               "META().id",
+		IndexSyncUser:               "META().id, name, email, disabled",
+		IndexSyncSession:            "META().id, username",
+		IndexSyncRole:               "META().id",
+		IndexSyncRoleExcludeDeleted: "META().id, deleted",
```

Comment:
We should just include the deleted property in IndexSyncRole and just use a single index for all role use cases.

Reply:
I tried that but with `IndexSyncRole` index, the query performs fetch on user.